BankSDK/GiniBankSDKExample/GiniBankSDKExampleUITests/GiniBankSDKExampleUITests.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  GiniBankSDKExampleUITests.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import GiniCaptureSDK
+import XCTest
+
+class GiniBankSDKExampleUITests: XCTestCase {
+
+    var app: XCUIApplication!
+    
+    override func setUpWithError() throws {
+    
+        try super.setUpWithError()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app.launchArguments = ["testing"]
+        app.launch()
+        print("we launch app")
```

Comment:
[clarification]: do we need this print here?

Reply:
No, we don't need it.